webpack.config.js

```diff
@@ -55,4 +57,13 @@ module.exports = {
         new webpack.optimize.UglifyJsPlugin(),
         new webpack.optimize.DedupePlugin(),
     ],
+    module: {
+        loaders: [
+            {
+                test: /\.js$/,
+                exclude: /(node_modules|bower_components|javascripts-legacy|components|vendor)/,
```

Comment:
It's a [Webpack thing](https://webpack.github.io/docs/configuration.html#module-loaders), rather than a Babel loader thing. You can pass an array of paths you want to `include`